app/models/image_file.rb

```diff
@@ -15,6 +15,17 @@ def process_image_data
   def file_format
     return unless file.attached?
 
-    errors.add(:image, "must be a JPEG, JPG, GIF, or PNG file") unless content_type.in?(%w[image/jpeg image/jpg image/gif image/png])
+    valid_types = %w[
+      image/jpeg
+      image/jpg
```

Comment:
[Only `image/jpeg` is a valid MIME type.](https://stackoverflow.com/questions/33692835/is-the-mime-type-image-jpg-the-same-as-image-jpeg#37266399) Let's remove `image/jpg`.